test/e2e/channel_subscription_ready_test.go

```diff
@@ -0,0 +1,31 @@
+// +build e2e
+// +build consolidated
+
+/*
+Copyright 2021 The Knative Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package e2e
+
+import (
+	"context"
+	"testing"
+
+	"knative.dev/eventing-kafka/test/e2e/helpers"
+)
+
+func TestChannelSubscriptionScaleReadyV1(t *testing.T) {
+	helpers.ChannelSubscriptionScaleReadyHelper(context.Background(), t, channelTestRunner)
```

Comment:
upstream is like:
https://github.com/knative-sandbox/eventing-kafka/blob/main/test/e2e/channel_subscription_ready_test.go#L30

Reply:
That skip is a later thing. Done by @devguyio in https://github.com/knative-sandbox/eventing-kafka/pull/573

Here, I'm only back porting knative-sandbox#476 as it required by knative-sandbox#563

You think we should backport https://github.com/knative-sandbox/eventing-kafka/pull/573 as well?

Reply:
There is also yours https://github.com/knative-sandbox/eventing-kafka/pull/685 @matzew, but you have abandoned it apparently :disappointed:

Reply:
I closed the PR, b/c not yet possible to remove the skip: https://issues.redhat.com/browse/SRVKE-772

Reply:
we need to still skip the test, so yeah we need to backport knative-sandbox/eventing-kafka#573 .

Reply:
I've added the knative-sandbox#573 commit as well.